packages/ceramic-cli/package.json

```diff
@@ -38,7 +38,8 @@
     "commander": "^5.0.0",
     "express": "^4.17.1",
     "identity-wallet": "1.2.0-ceramic.1",
-    "ipfs-http-client": "^44.0.0"
+    "ipfs-http-client": "^44.0.0",
+    "tweetnacl": "^1.0.3"
```

Comment:
Doesn't nodejs have a built in way to generate randomness? Seems exessive to import tweetnacl just for that.

Reply:
yep, but I thought there's a reason why the identity wallet uses `tweetnacl` for randomness. I'll change it! 👍

Reply:
I think IDW uses it because it can be run in browsers as well. IDW also uses tweetnacl for other things :)

Reply:
makes sense 👍